src/shared/ui/date-simple/date-simple.tsx

```
@@ -1,61 +1,66 @@
-      )}
-      {children}
-    </p>
-  );
-};
+import React from 'react';
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+
+import noteIcon from '@/shared/assets/icons/note-icon.svg';
+import { TO_BE_DETERMINED } from '@/shared/constants';
+import { dayJS } from '@/shared/helpers/dayJS';
```

ESLint

[error] There should be no empty line within import group

(import/order)